packages/account/src/Styles/account.scss

```diff
@@ -213,6 +213,18 @@ $MIN_HEIGHT_FLOATING: calc(
                 }
             }
         }
+        @include mobile-or-tablet-screen {
```

Comment:
let's just call it a responsive screen

Reply:
Hi @matin-deriv thanks for raising this but this is coming from Habib's PR https://github.com/binary-com/deriv-app/pull/15011/files

First they have to change then we can change 😬